src/API/API.ts

```diff
@@ -10,6 +10,7 @@ import { ExchangeConfiguration } from './ExchangeConfiguration'
 import { FileController, FileControllerConfiguration } from './FileController'
 import * as FileDAO from './FileDAO'
 import { HealthController } from './HealthController'
+import * as IPFSDirectoryHashDAO from './IPFSDirectoryHashDAO'
```

Comment:
can we avoid namespacing the import here?

Reply:
I was following Kenny's example. I think since the Symbols are no longer required, I can go back to a normal import

Reply:
I was just experimenting, sorry, didn't make for it to be taken as an example of how we are changing the code.

Reply:
Should experiments make it into production code? They become examples of how to do things.

Reply:
No, it was a mistake.